intelmq/bots/parsers/shadowserver/README.md

```diff
@@ -61,4 +61,11 @@ Every bot-type is defined by a dictionary with three values:
   tuple of showserver key and conversion function or not mention it at all.
   In both cases, the data will be added to extra.
 - `additional_fields`: A dictionary with a static mapping of field name to
-  data, e.g. to set classifications or protocols.
\ No newline at end of file
+  data, e.g. to set classifications or protocols.
+
+The tuples can be of following format:
+
+- `('intelmqkey', 'shadowkey')`
+- `('intelmqkey', 'shadowkey', conversion_function)`
+- `('shadowkey', conversion_function)`, data will be added to extra in this case
```

Comment:
An `extra_fields` dict may be confusing as all fields are added to extra, not only those mentioned there.

What about this:

    `('extra.', 'shadowkey', conversion_function)`

With the trailing dot it is clearly not a valid key, and thus more intuitive, that the field is added to extra using the conversion function.